common/kafka/kafka_watcher.cpp

```diff
@@ -423,6 +427,10 @@ void KafkaWatcher::StartWatchLoop() {
             // receiving ERR__PARTITION_EOF and there was still no messages to
             // be consumed after timeout.
           } else {
+            ++err_count_;
```

Comment:
same here.

Reply:
done

Reply:
in https://github.com/pinterest/rocksplicator/pull/377/commits/8524c2732077825e6063701eb9411ec9d8522728